### PR TITLE
Feat/improve docs

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: explanation.md [hunk 1]
```diff
@@ -1,74 +1,52 @@
-[Version française](./explanation_fr.md)
+[Lire en version française](./explanation_fr.md)
```
Comment (martinratinaud) [inline explanation.md]: Lire en français?

File: explanation.md [hunk 2]
```diff
@@ -1,74 +1,52 @@
...
 
 # Methodology
 
-Graphgenerator can recreate the network of interactions of Twitter accounts on a given hashtag or word (or group of words) 
-given. 
+Graphgenerator can recreate the network of interactions of Twitter accounts on a given hashtag or word (or group of words) given.
```
Comment (martinratinaud) [inline explanation.md]: Graphgenerator can recreate the network of interactions of Twitter accounts on a given hashtag or word (or group of words).

File: explanation.md [hunk 3]
```diff
@@ -1,74 +1,52 @@
...
-given. 
+Graphgenerator can recreate the network of interactions of Twitter accounts on a given hashtag or word (or group of words) given. 
 
 ## Collecting tweets
 
-The data is collected by `Graphgenerator` using the scraping tool `snscrape`. This tool collects the data
-that is displayed when you search using the Twitter search bar. It only allows to retrieve 
-the retweets of the last seven days (beyond that, only tweets and quotes can be collected). The 
-collection is limited to the last seven days.
+The data is collected by Graphgenerator using the scraping tool `snscrape`. This tool collects the data that is displayed when you search using the Twitter search bar. It only allows to retrieve the retweets of the last seven days (beyond that, only tweets and quotes can be collected). The collection is limited to the last seven days.
```
Comment (martinratinaud) [inline explanation.md]: Remove last sentence as it is redundant, and use bold

The data is collected by Graphgenerator using the scraping tool `snscrape`. This tool collects the data that is displayed when you search using the Twitter search bar. It only allows to retrieve the retweets of the **last seven days** (beyond that, only tweets and quotes can be collected).

File: explanation.md [hunk 4]
```diff
@@ -1,74 +1,52 @@
...
-To allow the good visualization of the graph, coordinates in a 2D (or 3D) plane are attributed to each 
-nodes.
+To allow the good visualization of the graph, coordinates in a 2D (or 3D) plane are attributed to each nodes.
 
-Several algorithms allow to compute the coordinates of the nodes of a graph to represent it as well as possible. 
-The objective is to summarize the information in the graph as well as possible while making it readable. 
-Generally the algorithms try to minimize the number of intersections of the links (for aesthetic considerations and to 
-ensure readability), to bring closer together the nodes that have many links between them and to place the "central" 
-nodes in the center of the graph (which allow "central" nodes (which allow information to circulate within the network).
+Several algorithms allow to compute the coordinates of the nodes of a graph to represent it as well as possible. The objective is to summarize the information in the graph as well as possible while making it readable. Generally the algorithms try to minimize the number of intersections of the links (for aesthetic considerations and to
```
Comment (martinratinaud) [inline explanation.md]: algorythms

File: explanation.md [hunk 5]
```diff
@@ -1,74 +1,52 @@
...
 
 ## Communities
 
-`Graphgenerator` also allows to identify clusters. They correspond to groups of accounts that interact 
-a lot with each other. 
+Graphgenerator also allows to identify clusters. They correspond to groups of accounts that interact a lot with each other. 
 
-Again, `networkx` provides a wide choice of algorithms for identifying these communities, most of which are 
-available in `graphgenerator`. 
+Again, Networkx provides a wide choice of algorithms for identifying these communities, most of which are available in Graphgenerator.
```
Comment (martinratinaud) [inline explanation.md]: algorythms

File: explanation.md [hunk 6]
```diff
@@ -1,74 +1,52 @@
...
-a lot with each other. 
+Graphgenerator also allows to identify clusters. They correspond to groups of accounts that interact a lot with each other. 
 
-Again, `networkx` provides a wide choice of algorithms for identifying these communities, most of which are 
-available in `graphgenerator`. 
+Again, Networkx provides a wide choice of algorithms for identifying these communities, most of which are available in Graphgenerator. 
 
-By default, we have chosen an algorithm that is not available in `networkx` but in an 
-library. It is based on the "Leuven method". This method is particularly efficient for large networks.
+By default, we have chosen an algorithm that is not available in Networkx but in an library. It is based on the "Leuven method". This method is particularly efficient for large networks.
```
Comment (martinratinaud) [inline explanation.md:41]: algorythm

File: explanation_fr.md [hunk 7]
```diff
@@ -1,85 +1,60 @@
-[English version](./explanation.md)
+[Read the english version](./explanation.md)
```
Comment (martinratinaud) [inline explanation_fr.md]: Read in English

**Review by martinratinaud — APPROVED** [review]
Some minor typos

Thanks for reviewing all this doc!

File: explanation.md (hunk 1 above)
Comment (clementbiron) [reply explanation.md]: Tu chipotes 🤣

**Pull request merged**